packages/rancher-logging/package.yaml

```diff
@@ -1,5 +1,5 @@
 url: https://kubernetes-charts.banzaicloud.com/charts/logging-operator-3.10.0.tgz
-packageVersion: 0
+packageVersion: 1
```

Comment:
I agree with bumping the packageVersion to 1 here (since a packageVersion of 0 doesn't sound right), but we shouldn't bump the packageVersion > once per release.

Reply:
Done